doc-requirements.txt

```
@@ -1,6 +1,6 @@
 # Doc and testing requirements
 sphinx
-myst-nb>=0.14
+git+https://github.com/jaladh-singhal/MyST-NB.git@677-scroll-output-config#egg=myst-nb
```

Lazy way of checking if the system allows non-existing config items or if we will have to bump the minimum requirements for the new myst-nb release.
```suggestion
myst-nb>=0.14
```